collectors/freebsd.plugin/freebsd_sysctl.c

```diff
@@ -2992,8 +2992,8 @@ int do_net_inet6_icmp6_stats(int update_every, usec_t dt) {
                 static RRDSET *st = NULL;
                 static RRDDIM *rd_in_errors = NULL, *rd_out_errors = NULL, *rd_in_csum_errors = NULL,
                               *rd_in_dest_unreachs = NULL, *rd_in_pkt_too_bigs = NULL, *rd_in_time_excds = NULL,
-                              *rd_in_parm_problems = NULL, *rd_out_dest_unreachs = NULL, *rd_out_time_excds = NULL,
-                              *rd_out_parm_problems = NULL;
+                              *rd_in_param_problems = NULL, *rd_out_dest_unreachs = NULL, *rd_out_time_excds = NULL,
+                              *rd_out_param_problems = NULL;
```

Comment:
I would leave the variables as they were. InParmProblems and OutParmProblems are consistent with the proc plugin names, the variables were named after them.

Reply:
https://iphostmonitor.com/mib/oids/ibm-icmpv6/ipv6IfIcmpInParmProblems.html

Reply:
Yeah, I consciously left the upstream object alone. The :eyes: are a mark reminding me to fix a thing.